### PR TITLE
UI : Add mlModel Entity Page

### PR DESCRIPTION
**Pull request by Sachin-chaurasiya**
See #1283 
### Describe your changes :

I worked on adding a machine learning model entity page.

Can access 
http://localhost:3000/mlmodel/{mlModelFqn}

#
### Type of change :

- [x] Improvement
- [x] New feature

#
### Frontend Preview (Screenshots) :

image

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/open-source-community/developer) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#
### Reviewers
@open-metadata/ui

### CODE REVIEW

**Comment by Sachin-chaurasiya** [comment]
Unit tests will be track by this PR https://github.com/open-metadata/OpenMetadata/issues/5072

**Comment by pmbrull** [comment]
On the first page, I believe that `Features` should be what we show, as that gives the most info about the model. Also, the differentiation point about our ML view is that `MlFeatures` have `FeatureSource`s which link a feature to a Table via an Entity Ref. We might need to show that as well

**Comment by pmbrull** [comment]
ooh the new features view looks great!

File: openmetadata-ui/src/main/resources/ui/src/pages/MlModelPage/MlModelPage.component.tsx [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+          throw jsonData['api-error-messages']['update-tags-error'];
+        }
+      })
+      .catch((err: AxiosError) => {
+        showErrorToast(
+          err,
+          jsonData['api-error-messages']['update-tags-error']
+        );
+      });
+  };
```
Comment (yug-shah0106) [inline openmetadata-ui/src/main/resources/ui/src/pages/MlModelPage/MlModelPage.component.tsx:177]: can we create util function something like this and use this across all API's? 
```
checkAPIError ( data:[any] , errorMessage:string){
      if(data.data > 0){
          return
      }
      else
        throw jsonData['api-error-messages'][errorMessage]
  }
```

we can use it like this

```
try{
const res = await   saveUpdatedMlModelData(updatedMlModel);
checkApiError(res,"string");
 setMlModelDetail((preVDetail) => ({
            ...preVDetail,
            mlFeatures: res.data.mlFeatures,
          }));
}
catch(err){
 showErrorToast(err);
}
```
@shahsank3t @vivekratnavel what your point on this ? any suggestion ?

Comment (vivekratnavel) [reply openmetadata-ui/src/main/resources/ui/src/pages/MlModelPage/MlModelPage.component.tsx:177]: Yes, we can add util and start using it everywhere.

Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/pages/MlModelPage/MlModelPage.component.tsx:177]: Thanks, @yug-shah0106 , I will make the required changes.

**Review by vivekratnavel — APPROVED** [review]

File: openmetadata-ui/src/main/resources/ui/src/components/MlModelDetail/MlModelFeaturesList.tsx [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+                        tagList={allTags}
+                        type="label"
+                        onCancel={() => {
+                          handleTagsChange();
+                        }}
+                        onSelectionChange={(tags) => {
+                          handleTagsChange(tags);
+                        }}>
+                        {feature.tags?.length ? (
+                          <button className="tw-ml-1 tw--mt-1 focus:tw-outline-none">
```
[the remark this answers is not in the document]
Comment (Sachin-chaurasiya) [reply openmetadata-ui/src/main/resources/ui/src/components/MlModelDetail/MlModelFeaturesList.tsx]: This is for a negative margin.

**Pull request merged**